src/pages/UserPage/UserPage.tsx

```diff
@@ -52,20 +52,20 @@ const UserPage: FC = () => {
   const { isLoading, isError, user } = useFetchUser({
     userId,
   })
+  const { existingRoles = [] } = useRolesFetch()
   const { archiveUser, isLoading: isArchiving } = useArchiveUser({
-    userId: userId,
+    institution_user_id: userId,
   })
-  const { existingRoles = [] } = useRolesFetch()
   const { activateUser, isLoading: isActivating } = useActivateUser({
     institution_user_id: userId,
   })
   const { deactivateUser, isLoading: isDeactivating } = useDeactivateUser({
     institution_user_id: userId,
   })
   const deactivationDate = user?.deactivation_date || ''
-  const forename = user?.user?.forename || ''
-  const surname = user?.user?.surname || ''
-  const name = `${forename} ${surname}`
+  const name = `${user?.user.forename} ${user?.user.surname}`
+
+  const isUserArchived = user?.archived_at !== null
```

Comment:
**[non-blocker]**

User actually also has the "status" field, which might be more reliable than checking whether "archived_at" date exists or not, but currently I'm not aware of any cases, where a user who isn't archived can have the "archived_at" value filled. Possibly in the future, when there will be some way to unarchive an account, but no need to deal with this now